src/interfaces/categories-data.ts

```diff
@@ -2,4 +2,12 @@ import { ICategoryItem } from './category-Item';
 
 export interface ICategoriesData {
   list: Array<ICategoryItem>
-}
\ No newline at end of file
+}
+
+export interface CategoryTableData {
+  id: number;
+  createdAt: string;
+  updatedAt: string;
+  name: string;
+  products: number;
+};
```

Comment:
please add new lines and othe identations

Reply:
+